.github/workflows/CI.yml

```diff
@@ -40,19 +41,20 @@ jobs:
       run: dotnet pack src/StackExchange.Redis/StackExchange.Redis.csproj --no-build -c Release /p:Packing=true /p:PackageOutputPath=%CD%\.nupkgs /p:CI=true
 
   windows:
-    name: StackExchange.Redis (Windows Server 2019)
-    runs-on: windows-2019
+    name: StackExchange.Redis (Windows Server 2022)
+    runs-on: windows-2022
     steps:
     - name: Checkout code
       uses: actions/checkout@v1
-    - name: Setup .NET Core 3.x
-      uses: actions/setup-dotnet@v1
-      with:
-        dotnet-version: | 
-          3.1.x
-          6.0.x
+    # - name: Install .NET SDK
+    #   uses: actions/setup-dotnet@v1
+    #   with:
+    #     dotnet-version: | 
+    #       6.0.x
+    - name: .NET Restore
+      run: dotnet restore Build.csproj --verbosity:diag
```

Comment:
this seems a little high for verbosity?

Reply:
Oh yeah sorry for context this is purely for a demonstration for https://github.com/NuGet/Home/issues/11548, never to merge! We can close this on out - will do so.